lib/tasks/server_recordings_sync.rake

```diff
@@ -0,0 +1,32 @@
+# BigBlueButton open source conferencing system - http://www.bigbluebutton.org/.
+#
+# Copyright (c) 2022 BigBlueButton Inc. and by respective authors (see below).
+#
+# This program is free software; you can redistribute it and/or modify it under the
+# terms of the GNU Lesser General Public License as published by the Free Software
+# Foundation; either version 3.0 of the License, or (at your option) any later
+# version.
+#
+# Greenlight is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License along
+# with Greenlight; if not, see <http://www.gnu.org/licenses/>.
+
+# frozen_string_literal: true
+
+desc 'Server Recordings sync with BBB server'
+
+task server_recordings_sync: :environment do
+  Recording.destroy_all
+
+  Room.select(:id, :meeting_id).in_batches(of: 25) do |rooms|
+    meeting_ids = rooms.pluck(:meeting_id)
```

Comment:
Can you not just do 
```
 Room.select(:meeting_id).in_batches(of: 25) do |rooms|
```

Reply:
No, all of the "batches" methods seem to return an ActiveRecord object.
Rails does not seem to provide a built-in way to [batch and pluck](https://gist.github.com/iamatypeofwalrus/f9f6a3049f63dd05d02c)

Reply:
There is a way to return the ids/index without firing an extra query https://stackoverflow.com/a/66885913